pages/dkp/kaptain/2.0.0/release-notes/2.0.0/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for Kaptain 2.0.0
+title: Release Notes for Kaptain 2.0.0
+menuWeight: 5
+beta: false
+excerpt: View release-specific information for Kaptain
+---
+
+**D2iQ&reg; Kaptain&reg; version 2.0.0 was released on April 25, 2022.**
+
+To get started with Kaptain, [download](../../download) and [install](../../install) the latest version of Kaptain.
+
+# Release Summary
+
+This release provides new features and capabilities that improve user experience, resolves reported issues, and changes the way Kaptain is deployed. This release supports Kubeflow 1.5 and a new version of the Kaptain SDK, 1.0.1. For more information on SDK 1.0.1, see [SDK release notes](../../sdk/1.0.x/release-notes).
+
+## New features and capabilities
+
+### Added Kaptain as DKP Catalog application
+
+Starting with Kaptain 2.0, it can be easily installed via the DKP catalog. [something something something]. Kaptain 2.0 is supported on DKP 2.1.2 onwards.
```

Comment:
```suggestion
Starting with Kaptain 2.0, it can be easily installed via the DKP workspace catalog in both single and multi-cluster experiences. Kaptain 2.0 is supported on DKP 2.1.2 onwards.
```